src/ocean/util/encode/Base64.d

```diff
@@ -553,6 +553,19 @@ unittest
         test!("==")(result, decode("TWE="));
         test!("!=")(result, decode(""));
     }
+
+    // don't decode short things at the end
+    {
+        auto result = cast(Const!(ubyte)[])"Maq";
+        // B should be ignored since it's missing the second byte
+        test!("==")(result, decode("TWFxB"));
```

Comment:
But it says just in the comment: "B should be ignored since it's missing the second byte". Anyway, I'll update, that's a fair suggestion.